### ISSUE TITLE
response to oomycete and innate immune response

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#19347]
We need to use both of these terms to say 
"oomycete innate immune response"

but these terms exist

GO:0061760    antifungal innate immune response  
GO:0140374    antiviral innate immune response  

if this is how we plan to do this, could we have
"oomycete innate immune response"
for consistency.

Not that, following this logic we will need these species terms  for every child in these branches

e.g
   plant-type antifungal  hypersensitive response  
   plant-type antioomycete hypersensitive response  

It doesn't seem very sustainable?

**Comment by ValWood** [comment]
@mgiglio99 
@dsiegele 

we touched on this issue tonight...

1. results in very inconsistent annotation
2. is unsustainable from an ontology development perspective

**Comment by pgaudet** [comment]
Is the proposal to obsolete ?

**Comment by ValWood** [comment]
That is a decision for the group.

We need to decide if we keep the response to terms and add the precomposed children, 
or obsolete....

**Comment by mgiglio99** [comment]
I haven't had a chance to think about this one yet. But I agree that making these taxon specific terms is unsustainable. Is the innate response to different microbes dependent on different proteins? Or do the same proteins protect against many microbes? I'm guessing the answer to that will be "sometimes it's one way and sometimes its another". Could we capture the taxon that the response is to with a dual taxon annotation?

Tagging @nsuvarnaiari to keep her in the loop

**Comment by ValWood** [comment]
> Could we capture the taxon that the response is to with a dual taxon annotation?

Yes we should be doing that anyway, but we should make it mandatory.
This would solve the problem. 
We would need to publicise and document  how to get this information from GO
